frontend-ssr/components/cli_git/match_commands.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+
+// class MatchWords extends React.Component {
+//     constructor(props) {
+//         super(props);
+
+//         this.state = {
+//             message: '',
+//         }
+//     }
+
+//     handleSubmit = (event) => {
+//         event.preventDefault();
+
+//         const inputValue = document.querySelector('[name="input_text"]').value;
+//         const {defaultValue} = this.props;
+
+//         if (Array.isArray(defaultValue)) {
+//             var result = defaultValue.some(
```

Comment:
Why 2 variables, `result` and `wordToMatch`? 

You should have just one, and depending on the situation (`Array` or `string`) you give it a certain value. Also, let's use `let` instead of `var`.